Modules/System/Assisted Setup/src/AssistedSetup.Codeunit.al

```diff
@@ -109,6 +109,15 @@ codeunit 3725 "Assisted Setup"
         AssistedSetupImpl.Open(AssistedSetupGroup);
     end;
 
+    /// <summary>Removes an Assisted Setup from a given extension so it will no longer be shown in the list.</summary>
+    /// <param name="ExtensionID">The app ID of the extension to which the setup belongs.</param>
+    /// <param name="PageID">The ID of the page to be removed.</param>
+    procedure Remove(ExtensionID: Guid; PageID: Integer)
```

Comment:
`ExtensionID` parameter isn't used.

Reply:
No, I used the Complete() and Run() procedures as template. I considered doing a check on the Assisted Setup record to ensure that you only delete your own.

Reply:
I have updated the pull request and omitted the ExtensionId parameter.

Reply:
We have removed this parameter from the mentioned functioned as they are not used :) (The changes are not yet uptaken in this repo).
So I advise you to do the same.

Reply:
I will update :) Thank you for the notice.